config/default/dblog.settings.yml

```diff
@@ -0,0 +1,3 @@
+row_limit: 1000
```

Comment:
shouldn't this be disabled and use syslog instead?

Reply:
with dblog disabled they don't get 404 reports. kinda dumb IMO